core/src/utils/CommonUtil.cpp

```diff
@@ -190,7 +190,7 @@ CommonUtil::GetExePath() {
     buf[cnt] = '\0';
 
     std::string exe_path = buf;
-    if (exe_path.rfind('/') != exe_path.length()) {
+    if (exe_path.rfind('/') != exe_path.length() - 1) {
```

Comment:
rfind invalid value should npos, please double check it.

Reply:
No, this judgement checks if the exe_path ends with '/'.